org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/PreferenceManager.java

```diff
@@ -104,6 +105,8 @@ public static void initialize() {
 		defEclipsePrefs.put(StubUtility.CODEGEN_EXCEPTION_VAR_NAME, "e"); //$NON-NLS-1$
 		defEclipsePrefs.put(StubUtility.CODEGEN_ADD_COMMENTS, Boolean.FALSE.toString());
 
+		defEclipsePrefs.put(CodeGenerationSettingsConstants.CODEGEN_USE_OVERRIDE_ANNOTATION, Boolean.TRUE.toString());
```

Comment:
I don't believe this is necessary. Override annotations are automatically added depending on the java version the project is compiling to. Eclipse default behavior just works IIRC

Reply:
OK. This is to make the test case work. I moved this into the setup code of the test case.